ha/util/consul_kv_store.py

```diff
@@ -120,9 +143,20 @@ def set(self, key: str, val: str=None):
             str: Return value.
         """
         self._verify_data(key)
+
         if self.key_exists(key):
             raise Exception(f"Key {key} already exists in kv store.")
-        self._consul.kv.put(self._prepare_key(key), val)
+
+        k = self._prepare_key(key)
```

Comment:
what is `k`? not very intuitive variable name.

Reply:
Hey, good input.
I do write long names but here it is already used a lot in this file so I just avoided changing to minimize the changesets, please let me know if I need to change.

Reply:
will fix in: https://jts.seagate.com/browse/CORTX-31319